src/utils/featureNotifications.tsx

```diff
@@ -0,0 +1,4 @@
+import { DateTime } from 'luxon';
+
+export const featureNotificationExpired = (preferenceTimestamp: number | undefined) =>
```

Comment:
will we use this on the org notification as well?

Reply:
Yes, didn't implement it yet